tests/phpunit/tests/test-rest-settings.php

```diff
@@ -163,6 +163,15 @@ public function test_unknown_method() {
 		$this->assertSame( 404, $response->get_status() );
 	}
 
+	public function test_get_settings_with_no_context() {
```

Comment:
This could go in `test_wrong_context()`.

Reply:
Not applicable anymore.